frame/support/tests/versioned_runtime_upgrade.rs

```diff
@@ -0,0 +1,243 @@
+// This file is part of Substrate.
+
+// Copyright (C) Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! Tests for VersionedRuntimeUpgrade
+
+#![cfg(feature = "experimental")]
```

Comment:
Could also opt to add `try-runtime` here, then there is no problem with the imports.

Reply:
Do you know if `try-runtime` tests are run in the CI? If so, then I see no issue with that

Reply:
Yea they should.